geonode/geoserver/manager.py

```diff
@@ -437,108 +442,109 @@ def set_permissions(
         approval_status_changed: bool = False,
         group_status_changed: bool = False,
     ) -> bool:
+
         _resource = instance or ResourceManager._get_instance(uuid)
 
         try:
             if _resource:
                 _resource = _resource.get_real_instance()
-                logger.error(
-                    f"Fixup GIS Backend Security Rules Accordingly on resource {instance} {isinstance(_resource, Dataset)}"
-                )
+                logger.info(f'Requesting GeoFence rules on resource "{_resource}" :: {type(_resource).__name__}')
                 if isinstance(_resource, Dataset):
-                    if settings.OGC_SERVER["default"].get("GEOFENCE_SECURITY_ENABLED", False):
+                    if settings.OGC_SERVER["default"].get("GEOFENCE_SECURITY_ENABLED", False) or getattr(
+                        settings, "GEOFENCE_SECURITY_ENABLED", False
```

Comment:
This is redundant. The `GEOFENCE_SECURITY_ENABLED `in the settings is used to populate the `OGC_SERVER["default"]["GEOFENCE_SECURITY_ENABLED"]`

Reply:
Nope, tests are only setting the `GEOFENCE_SECURITY_ENABLED` entry, not the other one